docs/how-to/core/use_images_in_prompts.md

```diff
@@ -0,0 +1,124 @@
+# How to define and use image prompts in Ragbits
+
+This guide will walk you through defining and using prompts in Ragbits that accept images as input. It covers handling single and multiple image inputs, incorporating conditionals in prompt templates based on the presence of images, and using such prompts with an LLM.
+
+## How to define a prompt with an image input
+
+### Using a single image as input
+
+To define a prompt that takes a single image as input, create a Pydantic model representing the input structure. The model should include an image field, which is either a **URL pointing to the image** or a **base64-encoded string**.
+
+```python
+import asyncio
+from pydantic import BaseModel
+from ragbits.core.prompt import Prompt
+from ragbits.core.llms.litellm import LiteLLM
+
+
+class ImagePromptInput(BaseModel):
+    """
+    Input model containing a single image.
+    """
+    image: bytes | str
```

Comment:
nit: maybe it would be better to use more specific example with field names different then `image` / `images`, so it is more clear that this is not some required name?